design-library/src/components/BccReact/BccReact.vue

```diff
@@ -56,8 +58,8 @@ function selectEmoji(emoji: EmojiStat) {
             </template>
           </TransitionGroup>
         </template>
-        <p v-else class="text-label flex items-center">
-          <KeyboardArrowLeftIcon class="mr-1 w-4" /> Be the first to react 😉
+        <p v-else class="bcc-react-empty">
```

Comment:
I suggest hiding the whole thing when the placeholder is empty:

```suggestion
        <p v-else-if="placeholder" class="bcc-react-empty">
```